From 10714f7af66e854e1e1c0b59cffe5e690bddd048 Mon Sep 17 00:00:00 2001
From: pmandri <padmashree.mandri@intel.com>
Date: Mon, 13 Dec 2021 12:43:05 +0530
Subject: [PATCH] Tuning period_size and period_count for QEMU/KVM

bsp_diff/common/vendor/intel/external/project-celadon/audio/01_679470_1-Tuning-period_size-and-period_count-for-QEMU-KVM.patch
This commit has dependancy on above patch.

Change-Id: Ib461120321a928f67352e4022ff8d6d7b403fbd7
Tracked-On:
Signed-off-by: pmandri <padmashree.mandri@intel.com>
---
 primary/audio_hw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/primary/audio_hw.c b/primary/audio_hw.c
index e4394c5..6db5c0d 100755
--- a/primary/audio_hw.c
+++ b/primary/audio_hw.c
@@ -50,7 +50,7 @@
 #define PCM_CARD_DEFAULT 0
 #define PCM_DEVICE 0
 
-#define OUT_PERIOD_SIZE 1024
+#define OUT_PERIOD_SIZE 4080
 #define OUT_PERIOD_COUNT 4
 #define OUT_SAMPLING_RATE 48000
 
-- 
2.17.1

